src/python/dxpy/scripts/dx.py

```diff
@@ -5543,6 +5547,8 @@ def __call__(self, parser, namespace, values, option_string=None):
                                 action='store_true')
 parser_new_project.add_argument('--database-ui-view-only', help='Viewers on the project cannot access database data directly', default=False,
                                 action='store_true')
+parser_new_project.add_argument('--monthly-compute-limit', type=positive_integer, help='Monthly compute limit for the project')
+parser_new_project.add_argument('--monthly-egress-bytes-limit', type=positive_integer, help='Monthly limit of egress bytes for the project')
```

Comment:
Monthly project spending limit for egress (in Bytes)

Reply:
done